File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int get_height(void);
void hash(int i);
void dot(int z);

int main(void)
{
    int a = get_height();
    for(int m=1; m<a+1; m++)
    {
        dot(a-m);
        hash(m);
        printf("  ");
        hash(m);
        printf("\n");
    }
}

int get_height(void)
{
    int p;
    do
    {
        p=get_int("Height: ");
    }
    while(p<1 || p>8);
    return p;
}

void hash(int i)
{
    for(int j=0; j<i; j++)
    {
        printf("#");
    }
}

void dot(int z)
{
    for(int x=0; x<z; x++)
    {
        printf(" ");
    }
}

File: pset6/mario.py
from cs50 import get_int

def main():
    while True:
        n = get_int("Height:")
        if n > 0 or n < 9:
            break

    for i in range(n):
        dot(n+1-i)
        hash(i+1)
        print("  ", end="")
        hash(i+1)
        print()


def hash(a):
    for i in range(a):
        print("#", end = "")
    return 0

def dot(b):
    for i in range(b):
        print(" ", end = "")

main()
